sql/catalyst/src/main/java/org/apache/spark/sql/streaming/KeyedStateTimeout.java

```diff
@@ -34,9 +32,20 @@
 @InterfaceStability.Evolving
 public class KeyedStateTimeout {
 
-  /** Timeout based on processing time.  */
+  /**
+   * Timeout based on processing time. The duration of timeout can be set for each group in
+   * `map/flatMapGroupsWithState` by calling `KeyedState.setTimeoutDuration()`.
+   */
   public static KeyedStateTimeout ProcessingTimeTimeout() { return ProcessingTimeTimeout$.MODULE$; }
```

Comment:
Its just that if someone this does `import KeyedStateTimeout._` the code boils down to 
`flatMapGroupsWithState(Update, ProcessingTime) { ... } ` with no reference to timeout. 

Fine either way.